data/operators/amenity/charging_station.json

```
@@ -6857,6 +6857,17 @@
         "operator": "ZES"
       }
     },
+    {
+      "displayName": "Zest",
+      "locationSet": {
+        "include": ["gb-eng", "gb-sct"]
```

Can we just do GB? It's only a matter of time until they have one in Wales.
```suggestion
        "include": ["gb"]
```